doc/LIMITS.md

```diff
@@ -233,14 +279,16 @@ const node = await createLibp2p({
      * Any connection with a `remoteAddress` property that has any of these
      * addresses as a prefix will be immediately rejected
      */
-     deny: [
+    deny: [
       '/ip4/132.14.52.64/tcp/3984',
       '/ip4/234.243.64.2',
       '/ip4/34.42',
       // etc
-    ]
-  }
-})
+    ],
+  },
+});
```

Comment:
More weird linting going on here, semi colons are disallowed in the main codebase:

```suggestion
})
```